src/main/flight/pid.c

```diff
@@ -547,43 +556,52 @@ rollAndPitchTrims_t *angleTrim, rxConfig_t *rxConfig)
 #endif
 
 #ifdef AUTOTUNE
-            if (shouldAutotune()) {
+            if (shouldAutotune())
+            {
                 error = DEGREES_TO_DECIDEGREES(autotune(rcAliasToAngleIndexMap[axis], &inclination, DECIDEGREES_TO_DEGREES(error)));
             }
 #endif
             PTermACC = error * (float)pidProfile->P8[PIDLEVEL] * 0.008f;
             tmp0flt = (float)pidProfile->D8[PIDLEVEL] * 5.0f;
```

Comment:
#526 is retaining the code style and should be at the same level than this one. tmp0 and tmp0flt variables are used in various locations in the code for different purposes? Not sure what a proper name could be. @ledvinap not sure if the incode declaration of the variables is more efficient if you do this multiple times?